angular/core/components/notifications/invitation_accepted.haml

```diff
@@ -1,11 +1,5 @@
 .media-left
   %user_avatar{user: "notification.actor()", size: "small"}
 .media-body
-  %span
-    %strong
-      %a.notifications__link{lmo-href-for: "notification.actor()"} {{notification.actor().name}}
-  %span{translate: "notifications.invitation_accepted"}
-  %span
-    %strong
-      %a.notifications__link{lmo-href-for: "notification.group()"} {{notification.group().name}}
+  %span{translate: "notifications.invitation_accepted", translate-value-name: "{{notification.actor().name}}", translate-value-group: "{{notification.group().name}}"}
```

Comment:
Ah, I guess not. I suppose we have empty `strong` tags floating around for a while there, heh.